lctool/run.py

```diff
@@ -37,6 +37,44 @@ def lcget():
                 nf.write(content.encode('utf-8'))
                 nf.write('\n')
 
+def lcrefresh():
```

Comment:
How about adding an argument such as '--override' to lc-get instead of making a new command since there are much overlap of the codes?